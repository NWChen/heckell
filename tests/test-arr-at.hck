let x: int array;
x = [1, 2, 3];

print(x.get(1));
print(x.get(2));

x.at(1, 500);
x.at(2, 40);

print(x.get(1));
print(x.get(2));