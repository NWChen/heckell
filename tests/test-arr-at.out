2
3
500
40
